commands/user.go

```diff
@@ -468,3 +490,106 @@ func verifyUserEmailWithoutTokenCmdF(c client.Client, cmd *cobra.Command, userAr
 	}
 	return nil
 }
+
+func userConvertCmdF(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	toBot, _ := cmd.Flags().GetBool("bot")
+
+	toUser, _ := cmd.Flags().GetBool("user")
+
+	if !(toUser || toBot) {
+		return fmt.Errorf("either %q flag or %q flag should be provided", "user", "bot")
+	}
+
+	if toBot {
+		return convertUserToBot(c, cmd, userArgs)
+	}
+
+	return convertBotToUser(c, cmd, userArgs)
+}
+
+func convertUserToBot(c client.Client, _ *cobra.Command, userArgs []string) error {
+	users := getUsersFromUserArgs(c, userArgs)
+	for _, user := range users {
+		if user == nil {
+			continue
+		}
+		bot, resp := c.ConvertUserToBot(user.Id)
+		if resp.Error != nil {
+			printer.PrintError(resp.Error.Error())
+			continue
+		}
+
+		printer.PrintT("{{.Username}} converted to bot.", bot)
+	}
+	return nil
+}
+
+func convertBotToUser(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	user := getUserFromUserArg(c, userArgs[0])
+	if user == nil {
+		return fmt.Errorf("could not find user by %q", userArgs[0])
+	}
+
+	password, err := cmd.Flags().GetString("password")
+	if err != nil {
+		return err
+	}
+
```

Comment:
We don't need to check the error here

```suggestion
	password, _ := cmd.Flags().GetString("password")
```